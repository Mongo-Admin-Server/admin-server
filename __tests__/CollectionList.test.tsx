/* eslint-disable no-undef */
import React from 'react';
import { render } from '@testing-library/react';
import CollectionList from '@/app/[locale]/components/ui/list/collection/CollectionList';

const mock = {
  collections: ['mock1', 'mock2']
};

describe('CollectionsList', () => {
  it('renders', () => {
    const { container } = render(<CollectionList collections={mock.collections} />)
    expect(container).toMatchSnapshot();
  })
})
