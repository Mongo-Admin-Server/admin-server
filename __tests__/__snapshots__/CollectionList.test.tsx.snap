// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CollectionsList renders 1`] = `
<div>
  <ul
    class="collection_list"
  >
    <li
      class="item "
    >
      mock1
    </li>
    <li
      class="item "
    >
      mock2
    </li>
  </ul>
</div>
`;
